event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/RunEventTransform.java

```diff
@@ -143,6 +145,31 @@ Payload buildRunPayload(Struct input) {
         payload.setCreatedAt(input.getString("created_at"));
         payload.setUpdatedAt(input.getString("updated_at"));
 
+        // If the option fields do not exist, set them to empty strings
+        try {
+            payload.setName(input.getString("name"));
+        } catch (DataException e) {
+            payload.setName("");
+        }
+
+        try {
+            payload.setWebConsoleUrl(URI.create(input.getString("web_console_url")));
```

Comment:
Note that these new fields are named inconsistently in the database (i.e. do not match the names used in the API). See https://github.com/RedHatInsights/playbook-dispatcher/blob/master/migrations/011_alter_runs_sat-fields.up.sql for how they are called in the database. The names you're using here in the model are correct, i.e. you'll need to e.g. take the `playbook_name` field value to populate the `name` model value.